ledger/common/catchpointlabel.go

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Affero General Public License
```

Comment:
this should probably be catchpoint subpackage or even go to `catchup/catchpoint`

Reply:
I'm not a big fan of that dependency graph direction. what do you think about moving it to `data/basics` ?